2024_homework/sorted_words2.rb

```diff
@@ -0,0 +1,18 @@
+# (2) Write a program called sorted_words.rb.  It should prompt the user for words and add each to an array.  The user should be able to add as many words as they like, until they just hit enter to return a blank word. Then sort the array using the sort method and print it out. 
```

Comment:
It's not explicitly stated in the question anywhere, but fun fact, `sort` doesn't deal with cases all too well. Test this out with a mix of uppercase and lowercase words and it probably won't work as expected.